WORKSPACE

```diff
@@ -555,6 +533,9 @@ hazel_repositories(
         {
             "z": "@com_github_madler_zlib//:z",
             "bz2": "@bzip2//:bz2",
+            "grpc": "@com_github_grpc_grpc//:grpc",
```

Comment:
Does this work in GHCi and on macOS? It looks like this is no longer using `fat_cbits`...

Reply:
Good question, it seemed to work on ghci on Linux but I might have had old files lying around. Let’s see what CI says.

Reply:
Looks like MacOS failed. I thought it also failed on Linux in the past so I’m a bit surprised but for now, I’ll go back to custom build files here.